var/spack/repos/builtin/packages/pmerge/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pmerge(AutotoolsPackage):
+    """PMERGE implements a new method that identifies candidate PSVs by
+       building networks of loci that share high levels of nucleotide
+       similarity. The PMERGE is embedded in the analysis pipeline of the
+       widely used Stacks software, and it is straightforward to apply it as
+       an additional filter in population-genomic studies using RAD-seq
+       data."""
+
+    homepage = "https://github.com/beiko-lab/PMERGE"
+    git      = "https://github.com/beiko-lab/PMERGE.git"
+
+    version('master',  branch='master')
+
+    depends_on('automake@1.14:1.14.99', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('m4', type='build')
+    depends_on('perl', type='build')
```

Comment:
The gitlab tests do not need to pass for this to be merged, but should this depend on `stacks` (actually looking at the PMERGE link, I think that would be `stacks@:1.48`).

Reply:
I'm not entirely clear!  
*My* reading of the pmerge page was that it *can* be used in a stacks pipeline.
It seems to build cleanly without stacks installed, but, again I'm not entirely clear on how it is supposed to work!
A user asked me to install it; I will check with them and see if they can give me some guidance on this.

Reply:
My user replied 'It should run independently of Stacks, but can be "inserted" into the stacks pipeline as an additional module compatible with stacks output down to version 1.48 or something like that.'.  Therefore, I don't think Stacks should be a dependency!